ocs_ci/ocs/utils.py

```diff
@@ -57,6 +58,15 @@
 log = logging.getLogger(__name__)
 
 
+mg_fail_count = 0
+mg_skip_count = 0
+mg_last_fail = None
+mg_collected_logs = 0
+mg_collected_types = set()
+mg_lock = threading.Lock()
```

Comment:
Delete as those are not relevant to this change and required change is not part of release-4.17 branch yet

Reply:
done